### PR TITLE
Create HarshDeep61034.json

### PR DESCRIPTION
**Pull request by HarshDeep61034**
Adding my Profile to LinkFree.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2986"></a>

### CODE REVIEW

File: data/HarshDeep61034.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+     {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/hdeep61034/?igshid=ZDdkNTZiNTM%3D",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/HarshDeep61034",
+      "icon": "FaGithub"
+    },
```
Comment (Aadarsh805) [inline data/HarshDeep61034.json:27]: Hey @HarshDeep61034 this comma at the end of line 27 needs to be removed

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/HarshDeep61034.json (hunk 1 above)
Comment (HarshDeep61034) [reply data/HarshDeep61034.json:27]: ok @Aadarsh805

**Comment by HarshDeep61034** [comment]
@ChinmayMhatre  plz check if there's anything wrong

**Review by Aadarsh805 — APPROVED** [review]
Everything looks good here

**Review by ChinmayMhatre — APPROVED** [review]
Looks Good!

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**